README.md

```diff
@@ -1,2 +1,33 @@
 # dataset-utils
 A collection of JavaScript functions to read and parse OpenActive Data Catalogs and Dataset Sites
+
+## Install
+
+Install using NPM:
+```
+$ npm install @openactive/data-catalog-crawler --save
```

Comment:
Assuming we want to stick with the rename?

Is mostly about the crawler as it turns out so we could also revert back to the `data-catalog-crawler` name. Just need to be consistent whichever we choose.

```suggestion
$ npm install @openactive/dataset-utils --save
```